modules/react/layout/stories/Box.stories.mdx

```diff
@@ -40,68 +10,21 @@ import {
 `Box` is a primitive component that provides a common, ergonomic API around Canvas design tokens. It
 is highly flexible, and its primary purpose is to build other components.
 
-## Table of Contents
+## Installation
 
-- [Box](#box)
-  - [Table of Contents](#table-of-contents)
-  - [Usage](#usage)
-  - [`As` Prop](#as-prop)
-  - [Ref](#ref)
-  - [Semantic Element Props](#semantic-element-props)
-  - [Style Props](#style-props)
-    - [Categories](#categories)
-    - [Border](#border)
-      - [Border Logical Props](#border-logical-props)
-      - [Border Shorthand Props](#border-shorthand-props)
-      - [Border Color Props](#border-color-props)
-      - [Border Radius Props](#border-radius-props)
-      - [Border Line Style Props](#border-line-style-props)
-      - [Border Width Props](#border-width-props)
-      - [Example](#example)
-    - [Color](#color)
-      - [Example](#example-1)
-    - [Depth](#depth)
-      - [Example](#example-2)
-    - [Flex Item](#flex-item)
-      - [Example](#example-3)
-    - [Font](#font)
-      - [Example](#example-4)
-    - [Layout](#layout)
-      - [Example](#example-5)
-    - [Position](#position)
-      - [Position Logical Props](#position-logical-props)
-      - [Position Standard Props](#position-standard-props)
-      - [Example](#example-6)
-    - [Space](#space)
-      - [Space Logical Props](#space-logical-props)
-      - [Space Standard Props](#space-standard-props)
-      - [Example](#example-7)
-    - [Text](#space)
-      - [Example](#example-8)
-  - [Exhaustive Style Prop List](#exhaustive-style-prop-list)
-    - [Border Props](#border-props)
-    - [Color Props](#color-props)
-    - [Depth Props](#depth-props)
-    - [FlexItem Props](#flexitem-props)
-    - [Layout Props](#layout-props)
-    - [Position Props](#position-props)
-    - [Space Props](#space-props)
+```sh
+yarn add @workday/canvas-kit-react
+```
 
 ## Usage
 
-```tsx
-import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+### Basic Example
```

Comment:
Now that you've updated the Box MDX to the standard MDX structure (thank you so much, by the way 👏), you 'll need to update the [Box](https://github.com/Workday/canvas-kit/blob/prerelease/major/.storybook/routes.js#L21) and [Flex](https://github.com/Workday/canvas-kit/blob/prerelease/major/.storybook/routes.js#L22) entries in the `routes.js` file to point to the new Basic stories for both components (`components-containers-layout-box--basic` and `components-containers-layout-flex--basic`). This should fix the [cypress error](https://dashboard.cypress.io/projects/odida5/runs/4544/test-results/60b75643-35ba-45b9-a3f7-16178f764b65?utm_source=github&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D) you're getting.

I assume the plan is to standardize the structure for `Flex.stories.mdx` as well in which case we would need to update the entry for Flex in the `routes.js` too.

Reply:
Yep! Thank you!